pkg/kubelet/dockertools/manager.go

```diff
@@ -2052,6 +2050,9 @@ func (dm *DockerManager) GetPodStatus(uid types.UID, name, namespace string) (*k
 		}
 		containerStatuses = append(containerStatuses, result)
 		if ip != "" {
+			if podStatus.IP != ip {
```

Comment:
My bad. I actually meant that comment for this PR in general. I will move my comment to `kubelet.go` changes.